pkg/metrics/chaos-daemon.go

```diff
@@ -59,7 +61,9 @@ type ChaosDaemonMetricsCollector struct {
 
 // NewChaosDaemonMetricsCollector initializes metrics for each chaos daemon
 func NewChaosDaemonMetricsCollector() *ChaosDaemonMetricsCollector {
+	logger, _ := z.NewDefaultZapLogger()
```

Comment:
ditto, please do not create another logger, acquire the logger as a parameter in the constructor.